server/schedulers/balance_test.go

```diff
@@ -848,6 +848,31 @@ func (s *testBalanceRegionSchedulerSuite) TestStoreWeight(c *C) {
 	testutil.CheckTransferPeer(c, sb.Schedule(tc)[0], operator.OpKind(0), 1, 3)
 }
 
+func (s *testBalanceRegionSchedulerSuite) TestIssue3544_UsageRatio(c *C) {
```

Comment:
I think a better naming should be TestUsageRatio and add comment `# Issue 3544`?